src/app/password-bars/password-bars.component.scss

```
@@ -0,0 +1,13 @@
+.strengthBlocks{
```

```suggestion
.strength-blocks {
```
name classes with `kebab-case`, check in other places